app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[show update]
+  before_action :authenticate_user!, only: %i[create update]
+
+  def index
+    @reservations = Reservation.all
+    render json: @reservations
+  end
+
+  # GET /reservations/1
+  def show
+    render json: @reservation
```

Comment:
don't you think we should also require only authenticated users to access this endpoint?

Reply:
:heavy_check_mark: Updated! 
```cmd
  before_action :authenticate_user!, only: %i[create update show]
```
:+1:

Reply:
Awesome!